src/main/java/core/basesyntax/model/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public interface GetArea {
+    double areaOfFigure();
```

Comment:
методы должны начинаться с глагола
```suggestion
    double calculateArea();
```